pupil-api/src/controllers/ping.ts

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+import { Request, Response } from 'express'
+import * as path from 'path'
+import * as fs from 'fs'
+import * as moment from 'moment'
+
+const getPing: any = async (req: Request, res: Response) => {
+  // get build number from /build.txt
+  // get git commit from /commit.txt
+  let buildNumber: object | string = 'NOT FOUND'
+  let commitId: object | string = 'NOT FOUND'
+  try {
+    buildNumber = await this.getBuildNumber()
```

Comment:
I think lines 14 & 20 should be without `this.` since they're declared as const now in the outer scope?

Reply:
Yes.  i think the whole file should be refactored into a class tbh.  Will tackle that this morning, if i have time.